panel/config.py

```diff
@@ -144,6 +144,9 @@ class _config(_base_config):
     throttled = param.Boolean(default=False, doc="""
         If sliders and inputs should be throttled until release of mouse.""")
 
+    load_entry_points = param.Boolean(default=True, doc="""
```

Comment:
Could we define the parameter in alphabetical order?

Reply:
np, done.